packages/gatsby-plugin-netlify/src/create-redirects.js

```diff
@@ -11,8 +11,29 @@ export default async function writeRedirectsFile(pluginData, redirects) {
   // Map redirect data to the format Netlify expects
   // https://www.netlify.com/docs/redirects/
   redirects = redirects.map(redirect => {
-    const status = redirect.isPermanent ? 301 : 302
-    return `${redirect.fromPath}  ${redirect.toPath}  ${status}`
+    const pieces = [
+      redirect.fromPath,
+      redirect.toPath,
+      redirect.isPermanent ? 301 : 302, // Status
```

Comment:
As a side note, for a future enhancement it would be good to support `200`, which serves the file without redirecting. A use case is for a simple single-page app where you want all URLs to go to the one actual HTML page. Also, `404`, which serves a custom 404 page for the URL (returns the provided HTML file with a `404` status code). Maybe instead of a boolean `isRedirect`, this could be a `mode` enum with `'temporaryRedirect'`, `'permanentRedirect'`, `'internalRedirect'` (200) or `'fileNotFound'`? Totally outside the scope of this PR though :)